requirements-dev.txt

```diff
@@ -8,4 +8,5 @@ rpm-py-installer
 docker
 -r requirements.txt
 -r https://github.com/ansible/ansible/raw/devel/test/lib/ansible_test/_data/requirements/sanity.pylint.txt
+-r https://github.com/ansible/ansible/raw/devel/test/lib/ansible_test/_data/requirements/sanity.rstcheck.txt
 -r https://github.com/ansible/ansible/raw/devel/test/lib/ansible_test/_data/requirements/sanity.validate-modules.txt
```

Comment:
Are you sure we still need those?
I don't see them in https://github.com/pulp/squeezer anymore.

Reply:
the validate-modules ones? maybe not. but didn't dare to try.